AerospikeClient/Async/AsyncCommand.cs

```diff
@@ -229,29 +241,60 @@ private void ExecuteCommand()
 			}
 			catch (AerospikeException.Connection aec)
 			{
+				exception = aec;
 				ErrorCount++;
 				ConnectionFailed(aec);
 			}
 			catch (AerospikeException.Backoff aeb)
 			{
+				exception = aeb;
 				ErrorCount++;
 				Backoff(aeb);
 			}
 			catch (AerospikeException ae)
 			{
+				exception = ae;
 				ErrorCount++;
 				FailOnApplicationError(ae);
 			}
 			catch (SocketException se)
 			{
+				exception = se;
 				ErrorCount++;
 				OnSocketError(se.SocketErrorCode);
 			}
 			catch (Exception e)
 			{
+				exception = e;
 				ErrorCount++;
 				FailOnApplicationError(new AerospikeException(e));
 			}
+			finally
+			{
+				if (activity != null && exception != null)
+				{
+					activity.SetStatus(ActivityStatusCode.Error, exception.ToString());
+				}
+			}
+		}
+
+		private void StartActivity()
+		{
+			// Because of the retry mecanism, one operation can result in several I/Os, on several nodes.
+			// For this reason, we create a new Activity on every retry, so that we can set the actual node
+			// adress and the I/O status as tags of this current Activity.
+
+			// If we're retrying, stop the previous Activity before restarting a new one.
+			activity?.Dispose();
+			activity = Tracing.Source.StartActivity(GetType().Name, ActivityKind.Client);
```

Comment:
https://opentelemetry.io/docs/specs/semconv/database/database-spans/

According to the spec, we should try to make the span name look like `<db.operation> <db.name>.<db.sql.table>`. Also this string should probably be cached as we don't want such allocation in the very hot path.